src/patternfly/components/Progress/progress.scss

```diff
@@ -139,7 +145,6 @@
   grid-column: 1 / 3;
   grid-row: 2 / 3;
   align-self: center;
-  height: var(--pf-c-progress__bar--Height);
-  background-color: var(--pf-c-progress__bar--BackgroundColor);
+  height: var(--pf-c-progress__indicator--Height);
+  background-color: var(--pf-c-progress__indicator--BackgroundColor);
 }
-
```

Comment:
can you add a blank line to the end of the file? We should have a blank line at the end of these files and most or all hbs files. Think of it as less of a blank line and more that you're putting a new line character after the end of the line above. This is one of the reasons you'll see odd indentation problems with hbs includes - if there is no newline character at the end of one partial, the include of another partial after it is included inline with the last line at the end of the previous one instead of starting on it's own line. If that makes any sense :)

Reply:
yes that makes sense! will look out for it!